modules/tensor_mechanics/doc/content/documentation/systems/Materials/ComputeStrainIncrementBasedStress.md

```diff
@@ -1,14 +1,33 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
+# Compute Strain Increment Based Stress
 
-# ComputeStrainIncrementBasedStress
+!syntax description /Materials/ComputeStrainIncrementBasedStress
 
-!alert construction title=Undocumented Class
-The ComputeStrainIncrementBasedStress has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
+## Description
 
-!syntax description /Materials/ComputeStrainIncrementBasedStress
+This stress calculator finds the value of the stress as a function of the elastic
+strain increment when a series of inelastic strains are specified in the input file.
+The stress is calculated as
+\begin{equation}
+  \label{eqn:stress}
+  \sigma_{ij} = \sigma_{ij}^{old} + C_{ijkl} \Delta \epsilon_{jk}^{el}
+\end{equation}
+where $\sigma_{ij}$ is the stress and $C_{ijkl}$ is the elasticity tensor of the
+material.
+The elastic strain increment, $\Delta \epsilon_{jk}^{el}$ is found by subtrating
+the sum of the inelastic strains from the mechanical strain:
+\begin{equation}
+  \label{eqn:elastic_strain_incr}
+  \Delta \mathbf{\epsilon}^{el} = \mathbf{\epsilon}^{mech} - \mathbf{\epsilon}^{mech-old}
```

Comment:
\mathbf -> \boldsymbol (\mathbf doesn't ever do anything to symbols, even when you're using LaTeX proper).